### PR TITLE
WIP: adding glitch icon

### PR DESCRIPTION
**Pull request by thisisjofrank**
I've made an SVG glitch icon (because i couldn't find a non-broken SVG glitch icon) but I don't know how to minimise it further and so can't get it below 5ish K, (nor do I know if that is possible!)

### CODE REVIEW

File: images/svg/glitch.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  <path fill="#f1a5ff" d="M424.27 330.76c-1.93-1.18-49.44-27.76-113.71-40.23-63.84-12.32-116.83-12-117.28-12a6 6 0 00-5.94 5.94 5.75 5.75 0 003 5 80.9 80.9 0 018.17 26.28c1.63 12.91-.15 27.17-1.79 33.1-.44 1.93-1 3.72-1.48 5.5h10.84a21 21 0 00.74-2.82 110.06 110.06 0 001.93-37 92.58 92.58 0 00-6.53-23.9 673.53 673.53 0 01106.14 11.73c62.2 12 109.7 38.6 110.15 38.89a5.17 5.17 0 003 .75 6 6 0 005.19-3c1.43-3 .54-6.75-2.43-8.24z"/>
+  <path fill="#373fff" d="M188.08 373.67c-.59 1.33-1 2.67-1.48 3.71a5.16 5.16 0 003 6.68 4.46 4.46 0 001.93.29 5.28 5.28 0 004.9-3.26c.44-1 .89-2.23 1.48-3.57a239.82 239.82 0 008.32-23.15h-10.87c-2.53 7.72-5.36 14.4-7.28 19.3z"/>
+  <path fill="#f1a5ff" d="M183.78 323.64c-.59-11.43-5.94-26-13.51-36.82a5.23 5.23 0 00-8.61 5.94c6.38 9.35 11.13 22.12 11.73 31.62.59 11.13-2.08 20.78-5.49 30.14h11c3.1-9.52 5.47-19.45 4.88-30.88z"/>
+  <path fill="#373fff" d="M168.93 366a4.5 4.5 0 001.93.29 5.29 5.29 0 004.9-3.26c1-2.82 2.23-5.79 3.27-8.76H168c-.59 1.63-1.18 3.26-1.93 4.9a5.09 5.09 0 002.86 6.83z"/>
+</svg>
```
Comment (AverageMarcus) [inline images/svg/glitch.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><defs><linearGradient id="a" x1="257.4" x2="257.4" y1="440.4" y2="306.1" gradientTransform="matrix(1 0 0 -1 0 511.9)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#e559ff"/><stop offset="1" stop-color="#924dff"/></linearGradient><linearGradient id="b" x1="167.6" x2="360" y1="364.5" y2="236.1" gradientTransform="matrix(1 0 0 -1 0 511.9)" gradientUnits="userSpaceOnUse"><stop offset=".1" stop-color="#f1a5ff"/><stop offset=".9" stop-color="#ff686c"/></linearGradient><linearGradient id="c" x1="257" x2="257" y1="267.7" y2="171.3" gradientTransform="matrix(1 0 0 -1 0 511.9)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#6f3cff"/><stop offset="1" stop-color="#3633ff"/></linearGradient><linearGradient id="d" x1="234.3" x2="325.8" y1="184.4" y2="74.8" gradientTransform="matrix(1 0 0 -1 0 511.9)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#f1a5ff"/><stop offset="1" stop-color="#ff686c"/></linearGradient></defs><rect width="512" height="512" fill="#fff" rx="76.8"/><path fill="url(#a)" d="M490.6 148.3c3.7-14.5 2.1-29.7 0-30.7-5.5-3.4-66.6 27.2-80.3 24.5-33.2-6.1-27.2-9.5-71.4-19.8-17.7-4-15-2-19-4.7a31.7 31.7 0 01-15.6-14.3C296.2 91.2 207.8 72 201.6 72c-12.2 0-35.3 31.3-35.3 31.3s-9 0-35.4 1.3-49.5 13-81.6 37.5c-32 24.6-27.2 49-27.2 49s26.9 8.7 29.1 14.2h257.1v-13.6l114.5-.6h63c-2.6-10.1-6.6-19.2-.5-29.3a51 51 0 005.3-13.5z"/><path fill="url(#b)" d="M485.3 214.2a29 29 0 002-7.4 38.7 38.7 0 00-1.6-15.6h-63a7.1 7.1 0 01-3.5 0H308.5v14.3H51.2a1.3 1.3 0 01.2.7 1.4 1.4 0 01-.2.8c-2 5.2-25.7 10.8-25.7 10.8 14.6 18.4 55.8 35.6 74.5 39.5l107 .6v-13.7h166.6v5.2c19.6-6 31.7-10.8 37.5-11.9 10.8-2.6 76.8 28.5 80.3 21.1a4.3 4.3 0 00.3-1.3l.1-1c-.1-8.2-14-23-6.5-42z"/><path fill="url(#c)" d="M244.5 340.1v14.3H50.6a3.3 3.3 0 00-.4-2.7c-1.7-2-6.1-4.3-13.8-7.3-7-2.8-14.9-5.6-14.9-5.6s-4.9-24.5 27.2-49a183.6 183.6 0 0148.5-29.4c2.3-.9 4.3-1.8 6.6-2.5l-3.7-.6 107 .6v-13.7h166.6v5.2a540.3 540.3 0 01-58.5 14c2 .8 3.4 2 4.7 2.7 3.9 2.5 1.6.9 16 4a28.5 28.5 0 013 .7c44.2 10.3 38.2 13.7 71.4 19.8 13.7 2.6 74.8-28 80.3-24.5 2.7 1.3 4.8 27.1-5.5 44.2-6 10.3-1.9 19.6.6 30h-61.4z"/><path fill="url(#d)" d="M491.4 401.7c-2.4-8.4-12.8-22-6.1-39.2 3.3-8.3 2.2-15.6.6-22.4H244.5v14.3h-194c-2 5.2-25.7 10.8-25.7 10.8 16.3 20.3 68 40 80.3 40a148 148 0 0038.8-2.6h2.6l-.7.7s12.2 28.6 38.7 34c13 3.5 26.6 3.5 40.1 1.4 4.8 0 9.5 0 14.3-1.4.7-7.4-.8-14.2-3.4-21 19 .7 37.4 0 58.5-1.4 70.6-10.2 106.1-26.5 117-29.2s76.8 28.5 80.3 21c.7-1.4.5-3 0-5z"/><path fill="#fff" d="M124.8 166a22 22 0 01-22 22 21 21 0 01-13.8-4.3 23.6 23.6 0 01-8.1-17.8c0-6.3 2.8-11.3 7-15.3s9-6.7 15-6.7a22.2 22.2 0 0122 22.1z"/><path fill="#020202" d="M115.3 165.8a17.2 17.2 0 01-17 17.4 19.6 19.6 0 01-11.9-4.2c-3.6-3.2-5-8-5-13.2 0-4.3 2-8 4.6-11a16 16 0 0112.3-6.3 17.2 17.2 0 0117 17.3z"/><circle cx="90.1" cy="172.4" r="5.5" fill="#fff"/><path fill="#fff" d="M124.8 315.3a22 22 0 01-22 22A21 21 0 0189 333a23.6 23.6 0 01-8.1-17.8c0-6.2 2.8-11.3 7-15.3s9-6.7 15-6.7a22 22 0 0122 22.1z"/><path fill="#020202" d="M115.2 315a17.2 17.2 0 01-17 17.4 19.6 19.6 0 01-12-4.2c-3.7-3-5-7.8-5-13.2 0-4.3 2.1-8 4.6-11a16 16 0 0112.4-6.4 17.2 17.2 0 0117 17.4z"/><circle cx="90.1" cy="321.6" r="5.5" fill="#fff"/><path fill="#f1a5ff" d="M424.4 180.5c-2-1.2-48.7-28.2-113-40.5-63.8-12.3-116.8-12-117.3-12a6 6 0 00-5.9 6 5.8 5.8 0 003 5 79.5 79.5 0 018.2 26.3 99.3 99.3 0 01-3.9 40.2h10.8c.5-1.5 1-3 1.2-4.5a110 110 0 002-37 92.6 92.6 0 00-6.6-23.8 673.4 673.4 0 01106.2 11.7A386 386 0 01418.3 191a5.2 5.2 0 003 .7 6 6 0 005.2-3 6.1 6.1 0 00-2-8.3z"/><path fill="#974dff" d="M189 223l-1.5 3.7a5.2 5.2 0 003 6.7 4.3 4.3 0 002 .3 5.3 5.3 0 004.9-3.3l1.4-3.5c2.3-5.4 5.2-13 7.9-21.7h-10.9c-2.5 7.3-5 13.3-6.8 17.8z"/><path fill="#f1a5ff" d="M184.5 173a77 77 0 00-13.5-36.8 5.2 5.2 0 00-8.6 6 70 70 0 0111.7 31.5 75.6 75.6 0 01-6 32h11a83 83 0 005.4-32.7z"/><path fill="#974dff" d="M169.7 215.4a4.5 4.5 0 002 .3 5.3 5.3 0 004.8-3.3l2.7-7H168a21 21 0 00-1.2 3.3 5.1 5.1 0 002.9 6.7z"/><path fill="#f1a5ff" d="M424.3 330.8c-2-1.2-49.5-27.8-113.7-40.3-63.9-12.3-116.9-12-117.3-12a6 6 0 00-6 6 5.8 5.8 0 003 5 80.9 80.9 0 018.2 26.2c1.6 13-.1 27.2-1.8 33.2-.4 1.9-1 3.7-1.5 5.5h10.9a21 21 0 00.7-2.9 110 110 0 002-37 92.6 92.6 0 00-6.6-23.9 673.5 673.5 0 01106.2 11.8 404.4 404.4 0 01110.1 38.9 5.2 5.2 0 003 .7 6 6 0 005.2-3c1.4-3 .5-6.7-2.4-8.2z"/><path fill="#373fff" d="M188 373.7l-1.4 3.7a5.2 5.2 0 003 6.7 4.5 4.5 0 002 .2 5.3 5.3 0 004.8-3.2l1.5-3.6a239.8 239.8 0 008.3-23.1h-10.8c-2.6 7.7-5.4 14.4-7.3 19.3z"/><path fill="#f1a5ff" d="M183.8 323.6a77 77 0 00-13.5-36.8 5.2 5.2 0 00-8.6 6 70.2 70.2 0 0111.7 31.6c.6 11.1-2 20.8-5.5 30.1h11a83.3 83.3 0 004.9-30.9z"/><path fill="#373fff" d="M169 366a4.5 4.5 0 001.9.3 5.3 5.3 0 004.9-3.3l3.2-8.7h-11l-2 4.9a5 5 0 003 6.8z"/></svg>
```
This gets it down to 4.7Kb

**Comment by edent** [comment]
This is a real tough one! I think the eyes can be duplicated. The gills (?) and side fin shapes can also be duplicated.  I think you could get away with repeating the bottom half of both fish.
I'm unsure if that'd be enough though.
I summon @judeGibbons !

**Comment by AverageMarcus** [comment]
4Kb on disk according to my Mac
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><defs><linearGradient id="a" x1="257.4" x2="257.4" y1="440.4" y2="306.1" gradientTransform="matrix(1 0 0 -1 0 512)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#e559ff"/><stop offset="1" stop-color="#924dff"/></linearGradient><linearGradient id="b" x1="167.6" x2="360" y1="364.5" y2="236.1" gradientTransform="matrix(1 0 0 -1 0 512)" gradientUnits="userSpaceOnUse"><stop offset=".1" stop-color="#f1a5ff"/><stop offset=".9" stop-color="#ff686c"/></linearGradient><linearGradient id="c" x1="257" x2="257" y1="267.7" y2="171.3" gradientTransform="matrix(1 0 0 -1 0 512)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#6f3cff"/><stop offset="1" stop-color="#3633ff"/></linearGradient><linearGradient id="d" x1="234.3" x2="325.8" y1="184.4" y2="74.8" gradientTransform="matrix(1 0 0 -1 0 512)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#f1a5ff"/><stop offset="1" stop-color="#ff686c"/></linearGradient></defs><rect width="512" height="512" fill="#fff" rx="76.8"/><path fill="url(#a)" d="M491 148c3-14 2-29 0-30-6-4-67 27-81 24-33-6-27-9-71-20-18-4-15-2-19-4a32 32 0 01-16-15c-8-12-96-31-102-31-13 0-36 31-36 31l-35 2c-26 2-50 13-82 37-32 25-27 49-27 49s27 9 29 14h257v-13l115-1h63c-3-10-7-19-1-29a51 51 0 006-14z"/><path fill="url(#b)" d="M485 214a29 29 0 002-7 39 39 0 00-1-16h-63a7 7 0 01-4 0H309v14H51a1 1 0 010 1 1 1 0 010 1c-2 5-26 11-26 11 15 18 56 35 75 39l107 1v-14h167v5l37-11c11-3 77 28 80 21a4 4 0 001-2v-1c0-8-14-23-7-42z"/><path fill="url(#c)" d="M245 340v14H51a3 3 0 00-1-2c-2-2-6-5-14-8l-15-5s-4-25 28-49a184 184 0 0148-30l7-2-4-1 107 1v-14h167v5a540 540 0 01-59 14l5 3c4 3 1 1 16 4a29 29 0 013 1c44 10 38 14 71 20 14 2 75-28 81-25 2 1 4 27-6 44-6 11-2 20 1 30h-62z"/><path fill="url(#d)" d="M491 402c-2-9-12-22-6-40 4-8 2-15 1-22H245v14H51c-2 6-26 11-26 11 16 21 68 40 80 40a148 148 0 0039-2h2s12 29 38 34c14 4 27 4 41 2 4 0 9 0 14-2 1-7-1-14-3-21 19 1 37 0 58-1 71-10 106-27 117-29s77 28 80 21v-5z"/><path fill="#fff" d="M125 166a22 22 0 01-22 22 21 21 0 01-14-4 24 24 0 01-8-18c0-6 3-11 7-15s9-7 15-7a22 22 0 0122 22z"/><path fill="#020202" d="M115 166a17 17 0 01-17 17 20 20 0 01-12-4c-3-3-5-8-5-13 0-4 2-8 5-11a16 16 0 0112-6 17 17 0 0117 17z"/><circle cx="90.1" cy="172.4" r="5.5" fill="#fff"/><path fill="#fff" d="M125 315a22 22 0 01-22 22 21 21 0 01-14-4 24 24 0 01-8-18c0-6 3-11 7-15s9-7 15-7a22 22 0 0122 22z"/><path fill="#020202" d="M115 315a17 17 0 01-17 17 20 20 0 01-12-4c-3-3-5-8-5-13 0-4 2-8 5-11a16 16 0 0112-6 17 17 0 0117 17z"/><circle cx="90.1" cy="321.6" r="5.5" fill="#fff"/><path fill="#f1a5ff" d="M424 181c-2-2-48-29-113-41-63-12-116-12-117-12a6 6 0 00-6 6 6 6 0 003 5 80 80 0 018 26 99 99 0 01-4 41h11l1-5a110 110 0 002-37 93 93 0 00-6-24 673 673 0 01106 12 386 386 0 01109 39 5 5 0 003 1 6 6 0 006-3 6 6 0 00-2-9z"/><path fill="#974dff" d="M189 223l-1 4a5 5 0 003 6 4 4 0 002 1 5 5 0 004-4l2-3 8-22h-11l-7 18z"/><path fill="#f1a5ff" d="M185 173a77 77 0 00-14-37 5 5 0 00-9 6 70 70 0 0112 32 76 76 0 01-6 32h11a83 83 0 005-33z"/><path fill="#974dff" d="M170 215a5 5 0 002 1 5 5 0 004-4l3-7h-11a21 21 0 00-1 4 5 5 0 003 6z"/><path fill="#f1a5ff" d="M424 331c-2-1-49-28-113-40-64-13-117-12-118-12a6 6 0 00-6 6 6 6 0 003 5 81 81 0 019 26c1 13-1 27-2 33l-2 5h11a21 21 0 001-3 110 110 0 002-36 93 93 0 00-7-24 674 674 0 01106 11 404 404 0 01110 39 5 5 0 004 1 6 6 0 005-3c1-3 0-7-3-8z"/><path fill="#373fff" d="M188 374l-1 3a5 5 0 003 7 5 5 0 002 0 5 5 0 004-3l2-4a240 240 0 008-23h-11l-7 20z"/><path fill="#f1a5ff" d="M184 324a77 77 0 00-14-37 5 5 0 00-8 6 70 70 0 0111 31c1 12-2 21-5 31h11a83 83 0 005-31z"/><path fill="#373fff" d="M169 366a5 5 0 002 0 5 5 0 005-3l3-9h-11l-2 5a5 5 0 003 7z"/></svg>
```

**Comment by thisisjofrank** [comment]
I've swapped it to the grayscale logo, and still can't get it below 1K!

**Comment by oliverturner** [comment]
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
  <symbol id="a">
    <ellipse cx="133.81" cy="301.72" fill="#fff" rx="11.05" ry="12.36"/>
  </symbol>
  <symbol id="b">
    <path fill="#fff" d="M188.07 310.36c-.91 12.93-2 17.53-6.28 27.33a7 7 0 01-6.36 4.11 6.78 6.78 0 01-2.8-.6 7 7 0 01-3.58-9.05c3-7.55 4.34-11 5.17-22.78.56-8.11-4-14.85-7.4-19.22a6.94 6.94 0 0110.87-8.63l.19.23c9.24 11.64 10.63 21.98 10.19 28.61z"/>
  </symbol>
  <symbol id="c">
    <path fill="#fff" d="M214.13 152.9a449.27 449.27 0 0157.42 5.79c27.49 4.82 59.19 15.23 62.7 16.4a6.47 6.47 0 01-2 12.6 6.37 6.37 0 01-2-.34c-.34-.1-33.64-11.15-60.84-15.92a437.19 437.19 0 00-55.41-5.61 6.46 6.46 0 11.14-12.92z"/>
  </symbol>
  <rect width="512" height="512" fill="#fff" rx="76.8"/>
  <path fill="#333" d="M428.68 306.41c-2.85-9.5-3.8-7.6 1.9-20.91a45.2 45.2 0 003.68-18 4.43 4.43 0 00-4.48-4.4 4.24 4.24 0 00-1.75.38c-8.73 3.79-25.38 11-33.56 14.4-11.41 4.75-23.76 6.65-38 1.9s-34.93-15.45-53.94-19.73c-12.72-2.86-20.87-5.36-34.12-6.8 2.37-.23 17.49-3.18 32.7-7.7a467.44 467.44 0 0044-16.14c17.1-7.61 28.51-5.71 40.85-1 7.48 2.88 21.88 9.57 32.12 14.42 4.11 1.94 10.09-1.86 8.12-5.95-3.64-7.51-4.59-20.35.16-28.9 5.4-9.72 4.28-11.87 1.43-21.37s-3.8-7.6 1.89-20.91a45 45 0 003.64-15.71 5.77 5.77 0 00-8-5.5c-9 3.93-24.09 10.42-31.74 13.61-11.4 4.75-23.76 6.65-38 1.9l-58-19.35a5.57 5.57 0 01-2.83-2.08c-2.51-3.55-9.31-12.42-16.11-14.69-8.55-2.85-39.91-12.33-51.31-14.25s-19-4.75-24.72 1.9c-4.21 4.93-10.9 12.89-14.12 16.73a5.77 5.77 0 01-4.69 2c-9.34-.43-40.33-.44-70.53 15.45-26.81 14.12-33.62 36.08-35.35 46.45a5.68 5.68 0 004.66 6.52h.09c8 1.27 22 4.3 21.82 9.75-.14 4.19-6.85 7.23-12.85 9.11a5.75 5.75 0 00-2 9.8c9.1 7.95 27.69 20.07 59.05 25.83a133.18 133.18 0 00-34.45 12.25C86.18 280.21 80 303.41 78.59 313.2a4.51 4.51 0 003.76 5.13c7.66 1.16 23.15 4.2 23 9.93-.15 4.67-8.45 7.9-14.8 9.7a4.45 4.45 0 00-1.86 7.55c10 9.31 34.59 26.82 81.55 30.27 7.2.53 15 .74 23 .73 6.83 9 19.14 21.62 37.15 25.82 13.63 3.19 22.06 2.47 27 1.05a5.39 5.39 0 003-8L247.23 374c11.12-.92 19-1.78 20.6-1.95a5 5 0 00.56-.11c4.4-1.1 61.22-15.43 77.66-22.74 17.11-7.6 28.51-5.7 40.87-1 8 3.09 24.12 10.61 34.38 15.49 3.35 1.58 7.4-1.43 6.26-4.94-2.63-8.16-3.57-23.36-.25-31 4.41-10.15 4.22-11.84 1.37-21.34z"/>
  <use href="#a"/>
  <use href="#a" transform="translate(0 -117)"/>
  <use href="#b"/>
  <use href="#b" transform="translate(0 -122)"/>
  <use href="#c"/>
  <use href="#c" transform="translate(0 115)"/>
</svg>
```
I've deduped some shapes, but I think @edent's right: making the body a symbol is the only way to get below 1Kb

**Comment by judeGibbons** [comment]
> This is a real tough one! I think the eyes can be duplicated. The gills (?) and side fin shapes can also be duplicated. I think you could get away with repeating the bottom half of both fish.
> I'm unsure if that'd be enough though.
> I summon @judeGibbons !

Funnily enough I have had a go at this one but I haven't looked at it for a long time. I've just checked and I've got a version at 1582 bytes and I think I got stuck after that!

**Comment by 17jiangz1** [comment]
I've managed to wrangle it to under 1024 bytes in #425, although the colours could possibly be improved to better represent the original icon within the limitations of solid colours.

**Comment by judeGibbons** [comment]
> I've managed to wrangle it to under 1024 bytes in #425, although the colours could possibly be improved to better represent the original icon within the limitations of solid colours.

I think I've managed a better one but I'm not sure of the best way to push it to the PR - do I need to fork your branch in addition to the fork I have of @edent's repo?

**Comment by edent** [comment]
@judeGibbons You can just send a separate PR - then I can compare them. Thanks all!

**Comment by edent** [comment]
Thanks all - have gone with Judy's one.

**Pull request closed without merging**